glfw.go

```diff
@@ -64,6 +64,23 @@ func (m *windowManager) sendPointerEvent(window *glfw.Window, phase embedder.Poi
 	}
 }
 
+func (m *windowManager) sendButtonPointerEvent(window *glfw.Window, phase embedder.PointerPhase, buttons embedder.PointerButtonMouse) {
```

Comment:
`sendPointerEventButton` would be more consistent with the `sendPointerEventScroll` below. But I have a feeling that this and the one below will cause conflicts when dragging AND scrolling. A sudden release would be caused because in `sendPointerEventScroll` the `Buttons` field isn't set.

Reply:
I think if Buttons is not passed in the struct the field is set to nil, then C embedder would set the value to null in C struct which is correct because when you scroll it's not a button.
Maybe I'm wrong and scroll is considered as 2 values. But I have no way to verify it.

Reply:
Buttons is of type PointerButtonMouse, which is a typed int64. So the zero-value ill be `0`, not `nil`.

What I mean is, when I *click* and drag and then scroll to drag my item further down, it creates the following series of events:
- PointerEvent with button set
- PointerEvent moving mouse,, button stil down
- PointerEvent with scroll vector < button is NOT set here, will be 0. Interpreted by flutter as a button release?
- (stopped scrolling, now moving mouse again) PointerEvent with button set < Interpreted as a new mouse-down?

Perhaps I should just setup a test scenario for this.. I haven't been able to try it for real yet.

Reply:
If value is 0 when no buttons is pressed then I can easily fix this problem.

Reply:
> PointerEvent with scroll vector < button is NOT set here, will be 0. Interpreted by flutter as a button release?

By reading the dart code, flutter has a special [case for handling scrolling AND dragging](https://github.com/flutter/flutter/blob/07aede4c3143b33fb7d3f871f67adcf7eed4c998/packages/flutter/lib/src/gestures/converter.dart#L423-L432). Flutter change the PointerPhase from PointerPhaseDown to PointerPhaseMove, if needed.

Recap:
 - PointerEvent with button set (PointerSignalKindNone, PointerPhaseDown, PointerButtonMousePrimary)
 - NO movement.
 - PointerEvent with scroll vector (PointerSignalKindScroll)
 - FLUTTER: Synthesize a hover/move of the pointer (need the PointerButtonMousePrimary)

Reply:
> PointerEvent with scroll vector < button is NOT set here, will be 0. Interpreted by flutter as a button release?

By reading the dart code, flutter has a special [case for handling scrolling AND dragging](https://github.com/flutter/flutter/blob/07aede4c3143b33fb7d3f871f67adcf7eed4c998/packages/flutter/lib/src/gestures/converter.dart#L423-L432). Flutter change the PointerPhase from PointerPhaseDown to PointerPhaseMove, if needed.

Recap:
 - PointerEvent with button set (PointerSignalKindNone, PointerPhaseDown, PointerButtonMousePrimary)
 - NO movement.
 - PointerEvent with scroll vector (PointerSignalKindScroll)
 - FLUTTER: Synthesize a hover/move of the pointer (need the PointerButtonMousePrimary)